.gitignore

```diff
@@ -11,3 +11,4 @@ phpunit.xml
 phpstan.neon
 /.phpunit.result.cache
 /docs/_build/
+.idea
```

Comment:
Cf discussion with @jordisala1991  here: https://github.com/sonata-project/dev-kit/pull/2043#discussion_r880931062

Please remove this change.